src/drivers/driver_zephyr.c

```diff
@@ -1546,6 +1596,170 @@ int wpa_drv_zep_deinit_ap(void *priv)
 
 	ret = 0;
 
+out:
+	return ret;
+}
+
+int wpa_drv_zep_sta_add(void *priv, struct hostapd_sta_add_params *params)
+{
+	struct zep_drv_if_ctx *if_ctx = NULL;
+	const struct zep_wpa_supp_dev_ops *dev_ops = NULL;
+	int ret = -1;
+
+	if ((!priv) || (!params)) {
+		wpa_printf(MSG_ERROR, "%s: Invalid params\n", __func__);
+		goto out;
+	}
+
+	if_ctx = priv;
```

Comment:
This looks like unnecessary assignment. We have already set the `if_ctx = NULL` earlier, so we could just set
`if_ctx = priv` at the beginning of the func.